dds/idl/be_util.cpp

```diff
@@ -35,12 +35,18 @@ be_util::prep_be_arg (char *arg)
   static const size_t SZ_WB_VERSIONING_END = sizeof(WB_VERSIONING_END) - 1;
   static const char WB_PCH_INCLUDE[] = "pch_include=";
   static const size_t SZ_WB_PCH_INCLUDE = sizeof(WB_PCH_INCLUDE) - 1;
+  static const char WB_CPP_INCLUDE[] = "cpp_include=";
+  static const size_t SZ_WB_CPP_INCLUDE = sizeof(WB_CPP_INCLUDE) - 1;
   static const char WB_JAVA[] = "java";
   static const size_t SZ_WB_JAVA = sizeof(WB_JAVA) - 1;
   static const char WB_TAO_INC_PRE[] = "tao_include_prefix=";
   static const size_t SZ_WB_TAO_INC_PRE = sizeof(WB_TAO_INC_PRE) - 1;
+  static const char WB_NO_DEFAULT_GEN[] = "no_default_gen";
```

Comment:
There are two different kinds of command-line arguments in `opendds_idl`.  The ones that are "top-level" like -L and -G options are handled separately (not in this file).  The others (in this file) are the -Wb, options.
As part of this change I think we should reconcile this and come up with a more systematic way of handling it.  Since we have -G options already, it would be good for rapidjson to be there, probably v8 too for consistency.  We can discuss more next week.

Reply:
Currently Tim's Options Are:
```
-Wb,cpp_include=<include path>
-Wb,no_default_gen
-Wb,rapidjson
```

So I'm thinking maybe
```
-Wb,cpp_include=<include path> (Stays the same)
-Sdefault-gen
-Grapidjson
```

In addition to those, changing these:
```
-Wb,java[=<output_file>]
-Wb,v8
-ZC <include>
```

To be consistent with the rest:
```
-Gjava[=<output_file>]
-Gv8
-Wb,ts_impl_cpp_include=<include path>
```

Reply:
- `-Sdefault-gen` could be just `-Sdefault`
- `-Grapidjson` OK
- `-Wb,java` I'd rather keep this way (or some other option) than use `-Gjava` which would be for actually generating the Java mapping (the current one is really a modifier to the type-support generator, though I'm not sure end users care)
- `-Gv8` OK
- `-Wb,ts_impl_cpp_include` reduce to `-Wb,ts_cpp_include` ?

For V8 we would probably keep compatibility with the old option.  Not sure we need the others, -ZC is an internal option for safety profile.